app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java

```diff
@@ -204,8 +204,13 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                */
 
               File shareFile = new File(shareFilePath);
-              Uri shareContentUri = FileProvider.getUriForFile(getContext(),
-                  BuildConfig.APPLICATION_ID + ".fileprovider", shareFile);
+              Uri shareContentUri;
+              if (Build.VERSION.SDK_INT >= 24) {
+                shareContentUri = FileProvider.getUriForFile(getContext(),
```

Comment:
I get a lint warning on this line. We intend to make our code base as clean as possible to remove all the lint warnings from this file. You can follow these steps:
- Open this file and go to menu **Analyze -> Inspect Code**, you will see a dialog window select the name of this file to inspect and press **OK**.
- An inspection result window will open, you have to select each warning one by one and see on the bottom right how to fix them. Wherever you have to pass a context you can pass `zimManageActivity` (like in this line). You can ignore spellings.
- After you have removed all the lint warnings, commit the changes using `git commit --amend` it will add these changes to your previous commit. Then force push the changes using `git push -f origin test`.

Reply:
@abdulwd I made the changes. But there were warnings before the addition of my lines.

Reply:
> @abdulwd I made the changes. But there were warnings before the addition of my lines.

@prajurock I know there must have been warnings but we want to get rid of them over a period of time.